prime-router/metadata/fhir_mapping/hl7/resource/DiagnosticReport.yml

```diff
@@ -200,4 +210,74 @@ extension_2:
         system: SYSTEM_URL, $system_code
         code: String, OBR.25
       constants:
-        system_code: v2-0123
\ No newline at end of file
+        system_code: v2-0123
+
+extension_3:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: parent-observation-sub-id
+    valueString:
+      type: STRING
+      valueOf: OBR.26.2
+      expressionType: HL7Spec
+
+extension_4:
+  generateList: true
+  expressionType: nested
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: parent-observation-value-descriptor
+    valueString:
+      type: STRING
+      valueOf: OBR.26.3
+      expressionType: HL7Spec
+
+extension_5:
+  condition: $systemCheck NOT_NULL
+  generateList: true
+  expressionType: nested
+  vars:
+    systemCheck: OBR.29.2
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: placer-assigned-identifier
+    valueIdentifier:
+      valueOf: datatype/Identifier_var
+      expressionType: resource
+      vars:
+        valueIn: OBR.29.1.1
+        systemCX: OBR.29.1.2
+        universalId: String, OBR.29.1.3
+        universalIdSystem: String, OBR.29.1.4
+      constants:
+        system: "http://terminology.hl7.org/CodeSystem/v2-0203"
+        code: "PLAC"
+        display: "Placer Identifier"
+
+extension_6:
+  condition: $systemCheck NOT_NULL
+  generateList: true
+  expressionType: nested
+  vars:
+    systemCheck: OBR.29.2
+  expressionsMap:
+    url:
+      type: SYSTEM_URL
+      value: filler-assigned-identifier
+    valueIdentifier:
+      valueOf: datatype/Identifier_var
+      expressionType: resource
+      vars:
+        valueIn: OBR.29.2.1
+        systemCX: OBR.29.2.2
```

Comment:
I can't get RMI to show up as OBR.29.2.2. I have tried adding type cast, among other iterations and it just won't show up.

Reply:
I think the changes I made in Luis's PR will fix this